src/Dockerfile.fedora

```diff
@@ -49,28 +49,25 @@ COPY root/usr/libexec/fix-permissions /usr/libexec/fix-permissions
 # safe in the future. This should *never* change, the last test is there
 # to make sure of that.
 {% if spec.version == "16" %}
-RUN INSTALL_PKGS="rsync tar gettext bind-utils postgresql16-server postgresql16-contrib nss_wrapper " && \
-    INSTALL_PKGS+=" procps-ng util-linux postgresql16-upgrade" && \
+RUN INSTALL_PKGS="rsync tar gettext bind-utils postgresql-server postgresql-contrib nss_wrapper " && \
+    INSTALL_PKGS+=" procps-ng util-linux postgresql-upgrade" && \
 {% else %}
-RUN INSTALL_PKGS="rsync tar gettext bind-utils postgresql-server postgresql-contrib nss_wrapper postgresql-upgrade procps-ng util-linux" && \
+RUN INSTALL_PKGS="rsync tar gettext bind-utils postgresql15-server postgresql15-contrib nss_wrapper postgresql15-upgrade procps-ng util-linux" && \
 {% endif %}
     INSTALL_PKGS+=" findutils xz" && \
-{% if spec.version not in ["9.6", "10", "11", "16"] %}
-    INSTALL_PKGS+=" pgaudit" && \
-{% endif %}
 {% if spec.version == "16" %}
-    INSTALL_PKGS+=" postgresql16-pgaudit" && \
-{% endif %}
-{% if spec.version not in ["16"] %}
-    dnf -y module enable postgresql:{{ spec.version }} && \
+    INSTALL_PKGS+=" pgaudit" && \
+{% else %}
+    INSTALL_PKGS+=" postgresql15-pgaudit" && \
```

Comment:
Same as above, `postgresql{{ spec.short }}` might be a bit more future looking.

Reply:
https://github.com/sclorg/postgresql-container/pull/583/commits/cb913c75bdea8a6f2ef634527e20dee1db14bf67